pkg/resource/resource.go

```diff
@@ -61,3 +58,8 @@ func MarshalYAML(r Resource) (interface{}, error) {
 		Spec:     r.Spec(),
 	}, nil
 }
+
+// String returns representation suitable for %s formatting.
+func String(r Resource) string {
+	return fmt.Sprintf("%s(%s/%s)", r.Metadata().Type(), r.Metadata().Namespace(), r.Metadata().ID())
```

Comment:
Maybe we can put Metadata in variable?
Like
```suggestion
	md := r.Metadata()
	return fmt.Sprintf("%s(%s/%s)", md.Type(), md.Namespace(), md.ID())
```